README.md

```diff
@@ -1,61 +1,59 @@
 # ansible-aci
 
-The ansible-aci project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
+## Description
 
-This collection has been tested and supports ACI 3.2+.
-Modules supporting new features introduced in ACI API in specific ACI versions might not be supported in earlier ACI releases.
+The `ansible-aci` project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
```

Comment:
I think the general norm is to expand an acronym the first time it is mentioned in a document after which only mentioning the acronym will do.


The `ansible-aci` project provides an Ansible collection for managing and automating your Cisco Application Centric Infrastructure (ACI) environment. It consists of a set of modules and roles for performing tasks related to ACI.

Reply:
Done 👍